packages/vue/src/components/atoms/SfDivider/SfDivider.stories.js

```diff
@@ -22,11 +22,11 @@ storiesOf("Atoms|Divider", module)
       template: `<SfDivider />`
     }),
     {
-     info: {
-       summary: `<p>Divider element.</p>
+      info: {
+        summary: `<p>Divider element.</p>
        <h2>Usage</h2>
        <pre><code>import SfDivider from "@storefrontui/vue/dist/SfDivider.vue"</code></pre>
```

Comment:
```suggestion
       <pre><code>import { SfDivider } from "@storefrontui/vue"</code></pre>
```

Reply:
I must've missed it previously

Reply:
uh, actually I edited it in this PR I just done:
https://github.com/DivanteLtd/storefront-ui/pull/216